addons/aushelper/bootstrap.js

```diff
@@ -35,16 +35,18 @@ const observer = {
       case "prefservice:after-app-defaults":
         TelemetryLog.log("WEBSENSE_DEFAULT_PREFS_RESET");
         break;
-      case "nsPref:changed":
+      case "nsPref:changed": {
         let branch = Services.prefs.getDefaultBranch("");
         let prefValue = branch.getCharPref(APP_UPDATE_URL_PREF);
         TelemetryLog.log("WEBSENSE_PREF_CHANGED", [prefValue]);
         break;
+      }
     }
   }
 };
 
-function startup() {
+/* eslint max-depth:off */
+function startup() { // eslint-disable-line complexity, max-statements
```

Comment:
The startup() function is too complex and needs breaking down - since I'm not actually working on the add-on, I didn't really want to do that work.